luamocks/SpringLuaApi.lua

```diff
@@ -70,6 +70,7 @@ Engine = {
 }
 
 ---@param path string
+--- @return any
 function VFS.Include(path)
```

Comment:
Can you revert this file? I understand this file is autogenerated.

Reply:
Reverting this line is easy. I don't know where the other changes even came from. It's a few dozen instances of removing the white space at the end of the line.

Out of curiosity, is there a way of setting sensible return types in that file? I think (because this was 2 months ago at this stage) the IDE was yelling at me a lot for mismatching type specs.

Reply:
I reverted the white space changes as well.

Reply:
FYI #3919 makes this change unnecessary.

Incidentally, I'd be interested to know how these files are autogenerated. I thought this particular file is hand-written but there is a subfolder of auto-generated files.